src/components/Select.vue

```diff
@@ -837,6 +853,7 @@ export default {
             blur: this.onSearchBlur,
             focus: this.onSearchFocus,
             input: (e) => (this.search = e.target.value),
+            mousedown: (e) => this.toggleDropdown(e),
```

Comment:
It is not clear to me, that is the purpose of `mousedown` here... Shouldn't it be covered by focus anyway, because we can open it by a keyboard as well as a mouse, and a mouse sets focus?

Reply:
At some point during testing this was needed to fix a bug but yes not neede anymore 😅